cloudflare.go

```diff
@@ -397,3 +397,18 @@ type WAFRulesResponse struct {
 		TotalCount uint `json:"total_count"`
 	} `json:"result_info"`
 }
+
+type PurgeZoneCacheRequest struct {
```

Comment:
I'm wondering if this should be merged with `PurgeCacheRequest` and set the `omitempty` tag. What do you think?